style.css

```diff
@@ -0,0 +1,229 @@
+}
+
+.nav a {
+	color: blanchedalmond;
+	background-color: transparent;
+	text-decoration: none;
+	font-family: "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+
+	Font-size: 13px;
+	Font-weight: 600;
+
+	Line-height: 18px;
+
+	color: rgba(255, 255, 255, 0.979);
+}
+.nav input {
+	position: absolute;
+	overflow: hidden;
+	width: 90px;
+	bottom: 3px;
+	height: 22px;
+	right: 15px;
+	border-radius: 10px;
+	background-image: linear-gradient(grey, #5f635f, grey);
+	border: 2px solid grey;
+}
+#apple {
+	background-image: linear-gradient(grey, #5f635f, grey);
+	color: #d7d8d9;
+}
+
+.nav i {
+	position: relative;
+	left: 3%;
+	top: 1.5px;
+	color: white;
+
+	z-index: 3;
+}
+
+.nav text {
+	background-image: linear-gradient(grey, #5f635f, grey);
+}
+
+.slogans {
+	text-align: center;
+	margin-top: 100px;
+	font-family: 'Raleway', sans-serif;
+	color: white;
+}
+
+#verse {
+	font-family: 'Raleway', sans-serif;
+	Font-size: 60px;
+	Font-weight: normal;
+	line-height: normal;
+	position: relative;
+	bottom: 50px;
+	color: white;
+}
+
+.slogans h2 {
+	Font-size: 30px;
+	Font-weight: 700;
+	Line-height: 18px;
+	color: white;
+	position: relative;
+	bottom: 36px;
+}
+
+.slogans h3 {
+	color: white;
+	font-size: 25px;
+}
+
+.slogans a {
+	text-decoration: none;
+}
+
+#play {
+	margin-left: 0.5%;
+}
+
+.bottom-images {
+	display: flex;
+	flex-direction: row;
+	margin-bottom: 0;
+	margin-top: 5px;
+	margin-left: auto;
+	margin-right: auto;
+}
+
+.bottom-images img {
+	width: 360px;
+	height: 200px;
+}
+
+footer {
+	display: flex;
+	flex-direction: row;
+}
+
+.footer-left {
+	justify-content: flex-start;
+}
+
+.apple-online {
+	justify-content: flex-start;
+	justify-content: space-evenly;
+	flex-direction: row;
+	Font-family: "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+	Font-size: 14px;
+	Font-weight: 400;
+	Line-height: 12.001px;
+	padding-top: 20px;
+	height: 14px;
+	position: relative;
+}
+
+.apple-online a{
+	text-decoration: none;
+	color: rgb(0, 136, 204);
+
+
+}
+
+.links {
+	display: flex;
+	justify-content: flex-end;
+	justify-content: space-evenly;
+	flex-direction: row;
+	Font-family: "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+	Font-size: 14px;
+	Font-weight: 400;
+	Line-height: 12.001px;
+	position: relative;
+	left: 40px;
+	padding-top: 20px;
+	height: 14px;
+}
+
+.links a{
+	text-decoration: none;
+}
+
+.links p {
+	padding-right: 10px;
+	padding-left: 10px;
+	border-right: 1px solid grey;
+	color: rgb(0, 136, 204);
+
+
+}
+
+.links img {
+	height: 14px;
+	margin-left: 10px;
+}
+
+.copyright{
+	justify-content: space-evenly;
+	flex-direction: row;
+	Font-family: "Lucida Grande", "Lucida Sans Unicode", Helvetica, Arial, Verdana, sans-serif;
+	Font-size: 14px;
+	Font-weight: 400;
+	Line-height: 12.001px;
+	padding-top: 20px;
+	height: 14px;
+}
+
+#terms{
+padding-right: 5px;
+border-right: 1px solid grey;
+color: rgb(0, 136, 204);
+

```

Comment:
This is one of the examples of the linting errors found.
Indentation problem. 
There were quite a few